### ISSUE TITLE
Errata from Paul Ellis

### ISSUE THREAD

**Issue by dcernst** [dcernst/IBL-IntroToProof#58]
This list references a PDF from 1/24/17.  That PDF is available [here](http://danaernst.com/resources/IntroToProof-Ellis.pdf).

  1. 2.40  I added the definition:  “The inverse of A->B is notA->notB”
  2. 2.46,2.47 The students need to assume that every integer is either even or odd.
  3. 2.92  These statements are equivalent in Z, but they are also equivalent in R, as they are both false in R.  I think you mean something a bit more subtle.
  4. 3.43: This is not a paradox if we are allowed $x\in x$.  This is, of course, not allowed in ZFC, but this is not clear outside of an axiomatic system.
  5. 3.63: "by the previous example" should read "by the previous axiom"
  6. 3.73, 3.97, 3.98:  I would write "The set of real numbers is...."  I think the way you wrote it may confuse some students, who are still, at this point, confusing properties of sets and elements.
  7. 3.90  This is way harder than it looks.
  8. Section 5.1: I gave a mini-lecture about "existence proofs".  It is a new "proof-skeleton"! [Assume x and y are both widgets, then prove that x=y] I then added the following exercise, which I borrowed from a textbook: "Uniqueness exercise" Assume c,a,r are real numbers so that c\neq 0, and r\neq a/c.  Then there exists a unique real x such that (ax+1)/(cx)=r 
  9. Remark 6.29: "these conditions are mutually exclusive" should read "these conditions are independent"
  10. Exercise 6.37: Instead of \Omega, I think you mean [x].
  11. Definition 6.49.  I would list "nonempty" as a separate property in the list.  It is a third thing they have to check in proofs, after all.
  12. Example 6.51.  I would write $\Omega=\{  \{a\}, \{b,c,d\}, \{e,f\}  \}$.  
  13. Theorem 6.83.  It was not clear to the students what to prove here, since it is a rewording of the definition.
  14. Theorem 6.88.  Ditto.  In fact, this is what I would put for the formal definition.

**Comment by dcernst** [comment]
Here's my current take:

  1. This seems reasonable to add.  Todo.
  2. Already done. I had already added the following comment to Chapter 2: "For the remainder of this section, you may assume that every integer is either even or odd but never both."
  3. Just fixed.
  4. Todo.
  5. Already done.
  6. Just fixed.
  7. Todo. Maybe add a hint?
  8. Todo.
  9. Already done.
  10. Already done.
  11. Already done.
  12. Already done.
  13. I've done some retooling in this section. For now, I'm ignoring this.

In summary, Items 1, 4, and 8 are outstanding.

**Comment by dcernst** [comment]
By the way, some of these were addressed in Issue #30.

**Comment by dcernst** [comment]
Item 1 is resolved.  Items 4, 7, 8 are still outstanding.

**Comment by dcernst** [comment]
My solution to item 4 was to remove the relevant problem. Items 7 and 8 are still outstanding.

**Comment by dcernst** [comment]
Item 7 is resolved.

**Comment by dcernst** [comment]
Item 8 is addressed.  I think that's all of them.
